k8s.gcr.io/images/k8s-staging-dns/images.yaml

```diff
@@ -90,6 +93,7 @@
     "sha256:974b9caa21ae26fb9857f3f80a434ccf05022ced51a2d84ff34528788eaf5331": ["1.22.9"]
     "sha256:b980edf0948382803d4e12f144eafa069cb1ea2b9b9ad0f365642167e5d5f762": ["1.22.10"]
     "sha256:ca0f273d55df875d82a586f0d9eee4915bd88d92af05e21305e4d00da73e79a6": ["1.22.11"]
+    "sha256:XXX": ["1.22.12"]
```

Comment:
could you fix that pls?

Reply:
sorry. fixed below
https://github.com/kubernetes/k8s.io/pull/4301/commits/5adea0139668b88f65da61ea72fedec461a7babb